client/src/ui/components/Spinner.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface SpinnerProps {
+  className?: string;
+  size: 'small' | 'large';
+}
+
+export const Spinner = ({ className, size }: SpinnerProps) => (
+  <React.Fragment>
+    <style>{`.loader {
```

Comment:
Using style like this is not the best way, we should rather use styled components in that case to define the animations :)